src/data_structures/message.cpp

```diff
@@ -1,25 +1,32 @@
+
+/** \brief Source file for message source code
+ * 
+ * Contains input and output stream methods from the header file
+ * which return float values of message structure
+ */
+
 #include <math.h> 
 #include <assert.h>
 #include <iostream>
 #include <fstream>
 #include <string>
-
-//updated relative path --Syed Omar
 #include "../../include/data_structures/message.hpp"
 
-/***************************************************/
-/************* Output stream ************************/
-/***************************************************/
-
+/**\brief Output stream
+ * 

```

Comment:
Can add detailed description or just have detailed without brief description

Reply:
Done. Thanks for your informative feedback